frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -454,6 +467,22 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
         ],
     }),
     listeners(({ props, values, actions, cache }) => ({
+        playerErrorSeen: ({ errorEvent }) => {
+            const fingerprint = encodeURIComponent(
+                errorEvent.message + errorEvent.filename + errorEvent.lineno + errorEvent.colno
+            )
+            if (values.reportedReplayerErrors.has(fingerprint)) {
+                return
+            }
+            captureException(errorEvent.error, {
+                extra: { fingerprint, recordingId: values.sessionRecordingId },
```

Comment:
```suggestion
                extra: { fingerprint: fingerprint, recordingId: values.sessionRecordingId },
```

unless this is implicit? since its a key:value

Reply:
yep, it's implicit... it's one of the nicest features of JS IMHO. `{fingerprint}` and `{fingerprint: fingerprint}` are equivalent